packages/tables/resources/views/components/summary/index.blade.php

```diff
@@ -24,10 +24,13 @@
             ->all();
     }
 
-    $hasPageSummary = (! $groupsOnly) && $records instanceof \Illuminate\Contracts\Pagination\Paginator && $records->hasPages();
+    $table = $this->getTable();
```

Comment:
Instead of calling methods on this object, please pass through the new parameters as attributes to the component like the rest